examples/miscellaneous/range-spectrogram.py

```diff
@@ -29,7 +29,7 @@
 __author__ = 'Alex Urban <alexander.urban@ligo.org>'
 
 # First, we need to load some data. As before we can `fetch` the
-# `public data <https://www.gw-openscience.org/catalog/>`__
+# `public data <https://gwosc.org/catalog/>`__
```

Comment:
I don't know if it matters, since this is a comment, but neither the original URL ([https://www.gw-openscience.org/catalog/](https://www.gw-openscience.org/catalog/)) nor the modified URL ([https://gwosc.org/catalog/](https://gwosc.org/catalog/)) works.

Reply:
Good catch, I'll fix that later...